content/pages/kits.md

```diff
@@ -0,0 +1,72 @@
+Title: Ordering, Building, and Operating DIYnamics Kits
+Slug: kits
+
+## The DIYnamics Kits
+
+The DIYnamics Project provides the resources for you to order, build and operate your own rotating tank platform! As detailed in the [About](/pages/about.html) page, these devices are used to model geophysical fluid dynamics. Rotating tanks of water feature fluid behavior analogous to planetary fluid bodies fundamentally impacted by rotation. As part of our Do-It-Yourself approach, our goal is for educators to perform experiments with rotating tank platforms themselves. Hands-on demonstrations provide a valuable opportunity for students to ask questions and draw conclusions in linking experimental results to real world phenomena.
+
+The DIYnamics Kits provide robust rotating tank platforms at various price points. Previously, obtaining your own device required an often custom design and complicated construction process. We developed our family of models to provide cheap, simple and robust solutions for your own device. Each of the below models is well suited for performing fluid dynamics experiments with students. The rotating tank platforms in the hierarchy of models differ in their ability to scale up in terms of speed and rotation rate.
```

Comment:
My suggested revision:

The DIYnamics Kits provide rotating tank platforms at various price points, sizes, and complexities.  Each is well suited for performing fluid dynamics experiments with students or for public outreach events.